source/security/index.md

```diff
@@ -433,10 +433,16 @@ with other known dangerous classes or packages in your application.
 
 We additionally recommend enabling the following options (enabled by default in 7.0).
 
- * `struts.ognl.allowStaticFieldAccess=false` - static methods are always blocked, but static fields can also optionally be blocked
- * `struts.disallowProxyMemberAccess=true` - disallow proxied objects from being used in OGNL expressions as they may present a security risk
```

Comment:
I think it's best we push developers to use `struts.disallowProxyObjectAccess` rather than `struts.disallowProxyMemberAccess`